dependency_install_steps_by_platform/Ubuntu_18.04.x.md

```diff
@@ -2,46 +2,61 @@
 
 This document assumes you are starting from a fresh install of Ubuntu in the 18.04.x series.
 
-Most recently tested 2018-05-21 with the following ubuntu release:
+
+## Install Git, VirtualBox, and rng-tools
 
 ```
-$ lsb_release --description
-Description:	Ubuntu 18.04 LTS
+$ sudo apt install git virtualbox rng-tools
 ```
 
 
 
-## Install Git, VirtualBox, Ansible, GnuPG, and rng-tools
+# Install Vagrant 2.0.3 or higher
+
+As of this writing, the vagrant version that Ubuntu 18.04 uses is 2.0.2 so we suggest a later
+release in the 2.0.x series.
 
 ```
-$ sudo apt install git virtualbox ansible gnupg2 rng-tools
+wget -c https://releases.hashicorp.com/vagrant/2.2.4/vagrant_2.2.4_x86_64.deb
+sudo dpkg -i vagrant_2.2.4_x86_64.deb
+rm vagrant_2.2.4_x86_64.deb
 ```
 
+Most recently tested 2019-03-26 with the following vagrant release:
 
+```
+$ vagrant --version
+Vagrant 2.2.4
+```
 
-## Install Vagrant 2.0.3 or higher
 
-As of this writing, the vagrant version that Ubuntu 18.04 uses is 2.0.2 so we suggest a later
-release in the 2.0.x series.
+
+## Choice: Install Ansible via apt now or pip later
+
+You can install apt to a system-wide location using Ubuntu's apt tool, which will be a less current
```

Comment:
Small typo here: "You can install Ansible to a system..."

Reply:
good catch, thanks!